roles/nginxplus/files/ratelimit.html

```diff
@@ -73,18 +73,14 @@
       <div class="wrapper">
         <h1>You have sent too many requests!</h1>
         <p>
-          We have rate-limited your traffic temporarily. If you are trying to download a large amount of data, try our Open Dataset. If you think your traffic should be allowed, contact Library Support.
+          We have rate-limited your traffic temporarily. If you are trying to download a large amount of data, <a href="https://catalog.princeton.edu/dataset">try our Open Dataset</a>. If you think your traffic should be allowed, contact Library Support.
```

Comment:
`try our <a href="https://catalog.princeton.edu/dataset">Open Dataset</a>`

Reply:
Thanks! Addressed in 1c0fd539ba2f6e1bdf791b3a8d1ced2953d91474.